# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/ranger/hotfix-setup.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ranger-1.9.4/setup.py.vanilla	2026-10-06 18:35:33.818261993 +0300
+++ ranger-1.9.4/setup.py	2026-10-06 18:35:44.435923228 +0300
@@ -4,7 +4,10 @@
 
 from __future__ import (absolute_import, division, print_function)
 
-from distutils import log  # pylint: disable=import-error,no-name-in-module
+import logging
+
+log = logging.getLogger(__name__)
+
 from hashlib import sha512
 import os
 import shutil
